pkg/k8s/apis/cilium.io/register.go

```diff
@@ -15,6 +15,86 @@
 package ciliumio
 
 const (
-	// GroupName is the group name of the cilium network policy
-	GroupName = "cilium.io"
+	// CustomResourceDefinitionGroup is the name of the third party resource group
+	CustomResourceDefinitionGroup = "cilium.io"
+
+	// CustomResourceDefinitionVersion is the current version of the resource
+	CustomResourceDefinitionVersion = "v2"
+
+	// CustomResourceDefinitionSchemaVersion is semver-conformant version of CRD schema
+	// Used to determine if CRD needs to be updated in cluster
+	CustomResourceDefinitionSchemaVersion = "1.20"
+
+	// CustomResourceDefinitionSchemaVersionKey is key to label which holds the CRD schema version
+	CustomResourceDefinitionSchemaVersionKey = "io.cilium.k8s.crd.schema.version"
```

Comment:
At list these consts are bounded to a particular version so they should belong to at least `...cilium.io/v2/register.go`

Reply:
Done.